source/plugin/economy/using.rst

```diff
@@ -34,18 +29,24 @@ Example: Loading the EconomyService
 
 .. code-block:: java
 
-	import org.spongepowered.api.event.Listener;
-	import org.spongepowered.api.event.service.ChangeServiceProviderEvent;
 	import org.spongepowered.api.service.economy.EconomyService;
+	import org.spongepowered.api.Sponge;
 
-	private EconomyService economyService;
-
-	@Listener
-	public void onChangeServiceProvider(ChangeServiceProviderEvent event) {
-		if (event.getService().equals(EconomyService.class)) {
-			economyService = (EconomyService) event.getNewProviderRegistration().getProvider();
-		}
+	Optional<EconomyService> serviceOpt = Sponge.getServiceManager().provide(EconomyService.class);
+	if (!serviceOpt.isPresent()) {
+	    // handle there not being an economy implementation
```

Comment:
I have two quibbles here (unless they breed). Comments in code blocks don't get translated, and it's clumsy wording - how about "handle the absence of an economy implementation".